elasticdl/python/worker/master_client.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+from elasticdl.proto import elasticdl_pb2, elasticdl_pb2_grpc
+from elasticdl.python.common.tensor_utils import serialize_ndarray
+
+
+class MasterClient:
+    """MasterClient provides some APIs connect with the master
+    service via gRPC call.
+
+    Usage:
+        channel = elasticdl.python.common.grpc_utils.build_channel(
+            "localhost:50001")
+        mc = MasterClient(channel, work_id=0)
+        # get task unit from master service
+        mc.get_task(...)
+    """
+
+    def __init__(self, channel, worker_id):
+        """initialize a master client.
```

Comment:
"Initialize a master client". Capitalize the first letter in the docstring.  There are the same questions in the following docstrings.